crates/router/src/connector/fiserv.rs

```diff
@@ -112,26 +110,17 @@ impl
         req: &types::PaymentsCaptureRouterData,
         _connectors: &settings::Connectors,
     ) -> CustomResult<Vec<(String, String)>, errors::ConnectorError> {
-        let timestamp = SystemTime::now()
-            .duration_since(UNIX_EPOCH)
-            .into_report()
-            .change_context(errors::ConnectorError::RequestEncodingFailed)?
-            .as_millis()
-            .to_string();
+        let timestamp = time::OffsetDateTime::now_utc().unix_timestamp();
```

Comment:
Sorry I forgot to mention this earlier, could you please add a utility function in the `common_utils::date_time` module called `now_unix_timestamp()` which returns this value?